### DISCUSSION TITLE
Using Resnet encoder in Unet.

### DISCUSSION THREAD

**Discussion by ramchandracheke** [Project-MONAI/MONAI#3024] [Ideas]
Hi,

I am just curious how to use resnet encoder as a backbone in the UNet architecture using MONAI framework.

Thanks,
Ram

**Comment by wyli** [comment]
that's not implemented yet, but recently there is an efficientnet encoder backbone, it is an extension of the `EfficientNet` https://github.com/Project-MONAI/MONAI/blob/a0130d1a7540872ea0cd972e00e8f32e8d774a47/monai/networks/nets/efficientnet.py#L555
please see the usage [in the tutorial](https://github.com/Project-MONAI/tutorials/blob/86ed9c98d212f570ff9ba6dcae5684d72136e4ec/kaggle/RANZCR/4th_place_solution/models/seg_model.py#L164-L182).
